DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AllDelegates/_SearchableDelegateCard.cshtml

```diff
@@ -7,7 +7,8 @@
   <details class="nhsuk-details nhsuk-expander">
     <summary class="nhsuk-details__summary">
       <span class="nhsuk-details__summary-text" id="@Model.DelegateInfo.Id-name">
-        <span class="searchable-element-title" name="name">@Model.DelegateInfo.TitleName</span> <span class="searchable-element-candidate-number">(@Model.DelegateInfo.CandidateNumber)</span>
+        <span class="searchable-element-title" name="name">@Model.DelegateInfo.TitleName</span>

```

Comment:
The change includes the email in the `searchable-element-title` span, so the JS search and sort will start returning results based on the email, but the non-JS search won't take it into account. From reading the ticket, it feels to me like we just want to change the display without changing the search functionality. Thoughts on this?

Same applies to the `_AddGroupDelegateCard`